controllers/sriovnetwork_controller_test.go

```diff
@@ -342,6 +342,68 @@ var _ = Describe("SriovNetwork Controller", Ordered, func() {
 				MustPassRepeatedly(10).
 				Should(Succeed())
 		})
+
+		Context("When the SriovNetwork namespace is not equal to the operator one", func() {
```

Comment:
can you add a case where the user requests to create nad in a different ns than the current one ?
then check nad doesnt get created with a `Consistently` block ?